data/mods/Aftershock/maps/furniture.json

```
@@ -243,6 +243,34 @@
       ]
     }
   },
+  {
+    "type": "furniture",
+    "id": "f_cooling_pylon",
+    "name": "cryo pod",
+    "symbol": "#",
+    "description": "This super chilled pylon is so cold it seems to pull in warmth towards itself and create a light breeze of freezing air.",
+    "color": "light_blue",
```

```suggestion
    "color": "light_cyan",
```
Just  so they look different to the pods when you use ASCII.